### DISCUSSION TITLE
Missing Import on local package

### DISCUSSION THREAD

**Discussion by cemde** [microsoft/pylance-release#3503] [Q&A]
I wrote a package that I am testing locally, but Pylance has issues scanning its paths.

In folder A, I have the repo of package A which I install to my conda environment using `pip install -e path/to/A

When I create a new project B importing A, I get this error message from Pylance:

```
Import "a.module1" could not be resolved. Pylance[reportMissingImports]
```

However, the code runs fine. The module is properly installed in the environment, so `from a.module1 import foo` works perfectly fine.

What are the likely causes of this?

[FYI: I am using the newest conda, newest vscode, python 3.10 on the newest MacOS]

**Comment by cemde** [comment]
I figured out a solution:
```
pip install -e dir/to/A --config-settings editable_mode=strict
```

**Reply by debonte** [reply]
If other people run into this, there are docs about it [here](https://github.com/microsoft/pylance-release/blob/main/TROUBLESHOOTING.md#editable-install-modules-not-found).
